From 8c27bcaea0116247ee055c5481a63507de4fe6e4 Mon Sep 17 00:00:00 2001
From: Marco Corte <marco@marcocorte.it>
Date: Wed, 2 Jul 2014 17:49:34 +0200
Subject: [PATCH 18/20] MINOR: stats: fix minor typo in HTML page

There is a very small typo in the statistics interface: a "set" in
lowercase where allothers are uppercase "Set".
---
 src/dumpstats.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dumpstats.c b/src/dumpstats.c
index c8bac08..5365042 100644
--- a/src/dumpstats.c
+++ b/src/dumpstats.c
@@ -3710,7 +3710,7 @@ static void stats_dump_html_px_end(struct stream_interface *si, struct proxy *px
 			      "<option value=\"\"></option>"
 			      "<option value=\"ready\">Set state to READY</option>"
 			      "<option value=\"drain\">Set state to DRAIN</option>"
-			      "<option value=\"maint\">set state to MAINT</option>"
+			      "<option value=\"maint\">Set state to MAINT</option>"
 			      "<option value=\"dhlth\">Health: disable checks</option>"
 			      "<option value=\"ehlth\">Health: enable checks</option>"
 			      "<option value=\"hrunn\">Health: force UP</option>"
-- 
1.8.5.5

